### ISSUE TITLE
re

### ISSUE THREAD

**Issue by flh** [patoline/patoline#3]

**Comment by rlepigre** [comment]
You mean that the environment variable INSTALL_UNICODELIB_DIR (set by configure.ml) does not always coincides with the directory in which ocamlfind actually installs unicodelib? What kind of value do you have in INSTALL_UNICODELIB_DIR? And "ocamlfind printconf destdir"?

**Comment by flh** [comment]
"ocamlfind install" call in src/unicodelib/Rules.mk writes files in /usr/local/lib/ocaml/4.05.0/unicodelib

Yet, configure.ml guesses /usr/local/lib/ocaml/unicodelib and writes this value in INSTALL_UNICODELIB_DIR and UnicodeLibConfig.ml

**Comment by flh** [comment]
Actually, configure.ml reports this:
The findlib default location is /usr/local/lib/ocaml/4.05.0
but the selected libdir is /usr/local/lib/ocaml

configure.ml should probably choose the value reported by findlib instead of sticking to its hard-coded default.

**Comment by rlepigre** [comment]
I agree. It would probably be best if the `get_config` function (in `configure.ml`) guessed the configuration from `ocamlfind` instead of `opam` (and maybe check that both configurations agree).
